connector/src/main/java/org/geysermc/connector/network/translators/world/block/entity/EndGatewayBlockEntityTranslator.java

```diff
@@ -39,7 +41,10 @@
 public class EndGatewayBlockEntityTranslator extends BlockEntityTranslator {
     @Override
     public void translateTag(NbtMapBuilder builder, CompoundTag tag, int blockState) {
-        builder.put("Age", (int) ((long) tag.get("Age").getValue()));
+        Tag ageTag = tag.get("Age");
+        if(ageTag instanceof LongTag) {
```

Comment:
Please add a space after `if` to follow with the rest of the project: `if (age instanceof[...]`.

After that, this looks good to me!